runtime/litentry/src/migration/balances_transaction_payment.rs

```diff
@@ -64,7 +64,11 @@ where
 			account_info.data.free = account_info.data.free.saturating_mul(DECIMAL_CONVERTOR);
 			account_info.data.reserved =
 				account_info.data.reserved.saturating_mul(DECIMAL_CONVERTOR);
-			account_info.data.frozen = account_info.data.reserved.saturating_mul(DECIMAL_CONVERTOR);
+
+			// Runtime migration is wrong at first. The below code has been mistakenly applied to
```

Comment:
I think it's better to remove this comment.

Reply:
I might prefer keep it. Just as a reminder of history view purpose.

Reply:
Oh sorry. I should remove it for litentry and litmus.